queries/cpp/highlights.scm

```diff
@@ -70,6 +77,12 @@
 "operator" @function
 "static_assert" @function.builtin
 
+[
+ "<<" 
```

Comment:
In cpp these operators often overwritten and used like functions.

Reply:
So what? What has this to do with highlighting? I would expect that all operator symbols would just have the `@operator` highlighting. It shouldn't matter how they are used in that specific instance.